cmd/deploy-metering/main.go

```diff
@@ -18,9 +18,10 @@ import (
 )
 
 var (
-	cfg        deploy.Config
-	deployType string
-	logLevel   string
+	cfg            deploy.Config
+	deployType     string
+	logLevel       string
+	meteringCRFile string
```

Comment:
In the follow-up we might want another struct for containing all the paths to the files so we can have function which opens all files + this and returns all the objects in the struct.